pages/about.html

```diff
@@ -5,8 +5,54 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link rel ="stylesheet" href="../styles/about_styles.css">
 </head>
 <body>
-  
+  <section class="container">
+        <header>
+            <nav>
+              <a href="file:///Users/ariamyonas/Developer/projects/personal-portfolio-site/pages/index.html">Home</a>
+              <a href="file:///Users/ariamyonas/Developer/projects/personal-portfolio-site/pages/portfolio.html">Portfolio</a>
```

Comment:
To make these hyperlinks accessible to other folks who pull up your code, we can access them by their relative path, like this:

```
<a href="index.html">Home</a>
<a href="portfolio.html">Portfolio</a> 
```